src/main/java/net/atos/zac/app/planitems/converter/RESTPlanItemConverter.java

```diff
@@ -57,7 +57,7 @@ public RESTPlanItem convertPlanItem(final PlanItemInstance planItem, final UUID
         return restPlanItem;
     }
 
-    public RESTPlanItem convertPlanItem(final PlanItemInstance planItem, final UUID zaakUuid, final PlanItemParameters parameters) {
+    public RESTPlanItem convertPlanItem(final PlanItemInstance planItem, final UUID zaakUuid, final HumanTaskParameters parameters) {
```

Comment:
Moet deze methode niet heten convertHumanTask?

Reply:
Ik snap je punt. We zetten het om naar RESTPlanItem, welke gebruikt wordt voor zowel de acties als de taken. En deze methode zet dan de extra attributen voor een HumanTask. Dus het geeft je een RESTPlanItem terug, waarvan je weet dat hij geïnterpreteerd gaat worden als een HumanTask. Dus ik kan de methode hernoemen, zodat je weet dat hier een HumanTask naar RESTPlanItem omgezet wordt.

Reply:
Heb het hernoemt en ook even naar de onderliggende code gekeken. Hij wordt vanuit de frontend aangeroepen via de rest service. In de frontend wordt een controle gedaan dat het type een human task is. Dus heb vanaf dat punt de methoden allemaal hernoemt naar HumanTask.